src/index.html

```diff
@@ -10,7 +10,33 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      data-qa="big"
+      class="search-bar"
+      action="/"
+      method="get"
+    >
+      <input
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        class="search-imput-big"
+    >
```

Comment:
not fixed spacings. check other same cases
```suggestion
      <input
        type="text"
        placeholder="Try &quot;Los Angeles&quot;"
        data-qa="keypress"
        class="search-imput-big"
      >
```